composer.json

```diff
@@ -11,7 +11,9 @@
         }
     ],
     "require-dev": {
-        "bcgov/wordpress-scripts": "2.1.2"
+        "bcgov/wordpress-scripts": "2.1.2",
+        "phpunit/phpunit": "^9.6",
+        "yoast/phpunit-polyfills": "^2.0"
```

Comment:
we should think about including this isn the dependency of wordpress-scripts aka wordpress-utils